config.json

```diff
@@ -992,6 +992,18 @@
         "text_formatting"
       ]
     },
+    {
+      "slug": "hangman",
+      "uuid": "ab3f8bf4-cfae-4f7a-b134-bb0fa4fafa63",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
```

Comment:
Since this approach of programming is not well known, I would suggest a higher difficulty level of 8.